### ISSUE TITLE
About caffe pretrained model

### ISSUE THREAD

**Issue by primeI** [open-mmlab/mmdetection#1586]
tks for your ranking implementation first.

Could you please tell whether the img should be normalized to [0, 1] while using the caffe-pretrained model? When I only substract the means((102.9801, 115.9465, 122.7717)), I got bad results on COCO. For some reasons, I must rewrite the dataloader~

tks again~

**Comment by primeI** [comment]
I couldn't find the preprocess(only call a self.pipline) details for the input images while using caffe-pretrained model. Could anyone help me about the details?
